design/javamp-deprecation.md

```diff
@@ -0,0 +1,115 @@
+# Java Microprofile Application Stack Deprecation
```

Comment:
Global comment:  the stack name, in the yaml, isn't "Java Microprofile" but "Eclipse MicroProfileÂ®"   
(with that trademark char maybe not necessary here)

The java-openliberty stack name, is indeed "Open Liberty".

Reply:
Yeah, the stack-id is java-microprofile.  I'm inclined to leave as-is.